cdxev/__main__.py

```diff
@@ -712,6 +714,66 @@ def create_build_public_bom_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_init_sbom_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "init-sbom",
+        help=("Provides the first draft of an SBOM for manual completion."),
+        description=(
+            "Creates an initial SBOM draft, CycloneDX version1.6, "
+            "to work with, containing the most basic fields. "
+            "Optionally for some fields values can be submitted to the command. "
+            "Those fields are: "
+            "The name for one author of the SBOM (metadata.authors[0].name) "
+            "The name of the supplier of the software (metadata.component.supplier.name) "
+            "The name of the software (metadata.component.name) "
+            "The version of the software (metadata.component.version)."
```

Comment:
This description belongs to the documentation, not cli help text, nobody will read this here.